contrib/database/sql/sql_test.go

```diff
@@ -55,6 +58,20 @@ func TestMySQL(t *testing.T) {
 		},
 	}
 	sqltest.RunAll(t, testConfig)
+	// additional test for uint64 values, supported by go-sql-driver/mysql
```

Comment:
This test seems a bit "snuck in" and out of place here. Can it maybe either have its own test or make it part of the `RunAll` test suite?